### PR TITLE
Add support for 1.17/1.18/1.19

### PR DESCRIPTION
**Pull request by jknipper**
This PR adds support for k8s 1.17/1.18/1.19 to Kubernikus. It depends on #448 because node leases are now used.

### CODE REVIEW

File: charts/kube-master/templates/api.yaml [hunk 1]
```diff
@@ -189,7 +191,9 @@ spec:
             - --enable-admission-plugins=PersistentVolumeLabel
 {{- end }}
 {{- if (semverCompare ">= 1.14" .Values.version.kubernetes) }}
+{{- if (semverCompare "< 1.17" .Values.version.kubernetes) }}
```
Comment (databus23) [inline charts/kube-master/templates/api.yaml:194]: Interesting, why is this necessary?
This might conflict with hammertime which depends on frequent node updates to not kick in. 
In general we should switch to NodeLeases anyway but this requires updating the client-go in kubernetes.

Comment (jknipper) [reply charts/kube-master/templates/api.yaml:194]: Apparently this feature needs to be turned on:
```
error: invalid argument "NodeLease=false" for "--feature-gates" flag: cannot set feature gate NodeLease to false, feature is locked to true
```

**Comment by BugRoger** [comment]
Ressurect

**Comment by jknipper** [comment]
I'm going to update this to the latest 1.17 release.

**Review by databus23 — APPROVED** [review]
Looks good. Great work!
